charts/events/templates/event-bus.yaml

```diff
@@ -0,0 +1,36 @@
+# https://raw.githubusercontent.com/argoproj/argo-events/stable/examples/eventbus/native.yaml
+{{- if index .Values "argo-events" "enabled" }}
+  {{- range $eventBus := $.Values.eventBuses }}
+---
+apiVersion: argoproj.io/v1alpha1
+kind: EventBus
+metadata:
+  namespace: {{ $eventBus.namespace }}
+  name: {{ $eventBus.name }}
+spec:
+  nats:
+    native:
+      # Optional, defaults to 3. If it is < 3, set it to 3, that is the minimal requirement.
+      replicas: 3
+      # Optional, auth strategy, "none" or "token", defaults to "none"
+      auth: token
+      {{- if eq ($eventBus.environment | default "prod") "dev" }} 
```

Comment:
I would prefer if we took the individual values from the `values.yaml` and over-rid them with `dev-values.yaml`. This bakes the idea of `values` and `dev-values` into the chart, where they don't belong